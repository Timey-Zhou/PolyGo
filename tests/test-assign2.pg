void main()
{
	int i;
	float j;
	string s;
	bool b;
	i = 10086;
	j = 1.2345;
	s = "Hey";
	b = true;
	print(i);
	print_f(j);
	print_s(s);
	print_b(b);
}