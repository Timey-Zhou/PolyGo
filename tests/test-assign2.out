10086
1.234500
Hey
1
